Assets/Scripts/Models/Tile.cs

```diff
@@ -111,18 +112,16 @@ public float PathfindingCost
 
     public int Y { get; private set; }
 
+    public float MovmentModifier { get; set; }
```

Comment:
For that matter, why bother having {get; set;}? Either make it a protected set, or else just remove the accessors.